src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
```

Comment:
The variable 'total' is initialized but never changed; it remains 0. The assignment to 'total' is unnecessary. You can return the result of the calculation directly.